### ISSUE TITLE
StackingCVClassifier & StackingCVRegressor should use any cross validation strategy 

### ISSUE THREAD

**Issue by sque** [rasbt/mlxtend#202]
It would be better if `StackingCVClassifier` had the same interface as `cross_val_score` where instead of having `n_folds` and `stratify` a unified `cv` argument could control all options for cross validation strategy.

This opens the possibility to do the following:
```
# Use a default KFold, splits=3
clf = StackingCVClassifierEx(..., cv=3)

# Use a Stratified KFold, splits=3
clf = StackingCVClassifierEx(..., cv=StartifiedKFold(n_splits=3))

# Use a different cross validation with groups
clf = StackingCVClassifierEx(..., cv=GroupKFold(n_splits=3))

# Use a custom iteration
cv_iters = np.array([[...]])
clf = StackingCVClassifierEx(..., cv=cv_iters)
```

To implement this I propose to reuse the internal function `sklearn.model_selection._split.check_cv` which process the input of `cv` argument and returns a cross validation object that has `split()` method.

I am willing to open a pull request if you are ok with this change.

**Comment by rasbt** [comment]
That's a good idea @sque ! If you could create a PR with this, that would be great.

Also, there's the StackingCVRegressor about to be merged (#201) I suppose we could do the same thing here (although it wouldn't support stratification).

**Comment by rasbt** [comment]
adressed in #203 & #204
